ci-operator/step-registry/hypershift/mce/kubevirt/create/hypershift-mce-kubevirt-create-chain.yaml

```diff
@@ -45,12 +52,14 @@ chain:
 
 
         echo "$(date) Creating HyperShift cluster ${CLUSTER_NAME}"
-        /tmp/hypershift create cluster kubevirt ${ICSP_COMMAND} \
+        /tmp/${HYPERSHIFT_NAME} create cluster kubevirt ${ICSP_COMMAND} \
           --name ${CLUSTER_NAME} \
           --node-pool-replicas ${HYPERSHIFT_NODE_COUNT} \
           --memory 16Gi \
           --cores 4 \
           --namespace local-cluster \
+          --release-image ${RELEASE_IMAGE_LATEST} \
+          --etcd-storage-class gp3-csi \
```

Comment:
This will fail for `baremetalds` though since there will be no gp3-csi. maybe we should only do this if aws is in use as the underlying infra cluster